src/components/send/form/formBase.js

```diff
@@ -18,33 +19,54 @@ import links from '../../../constants/externalLinks';
 import regex from '../../../utils/regex';
 import styles from './form.css';
 
-function getInitialState() {
-  return {
-    isLoading: false,
-    fields: {
-      recipient: {
-        address: '',
-        error: false,
-        feedback: '',
-        selected: false,
-        title: '',
-        value: '',
-      },
-      amount: {
-        error: false,
-        value: '',
-        feedback: '',
-      },
-    },
-    unspentTransactionOutputs: [],
-  };
+function getAmountFeedbackAndError(value, {
```

Comment:
this method is completely independent of this component, so it should move to the validation utils file